README.md

```diff
@@ -760,7 +759,7 @@ We've talked with the Readonly Collections champions, and both groups agree that
 
 Neither one is a subset of the other in terms of functionality. At best, they are parallel, just like each proposal is parallel to other collection types in the language.
 
-So, the two champion groups have resolved to ensure that the proposals are in parallel *with respect to each other*. For example, this proposal adds a new `Tuple.prototype.withReversed` method. The idea would be to check, during the design process, if this signature would also make sense for read-only Arrays (if those exist): we extracted these new methods to the [Change Array by copy](https://github.com/tc39/proposal-change-array-by-copy/) proposal, so that we can discuss an API which builds a consistent, shared mental model.
+So, the two champion groups have resolved to ensure that the proposals are in parallel *with respect to each other*. For example, this proposal adds a new `Tuple.prototype.withReversed` method. The idea would be to check whether this signature also makes sense for read-only arrays (if those exists) during the design process: we extracted these new methods to the [Change Array by copy](https://github.com/tc39/proposal-change-array-by-copy/) proposal so that we can discuss an API for building a consistent, shared mental model.
```

Comment:
When "during the design process" is immediately attached to the initial clause, it is less ambiguous that it means the idea was to *check during the design process*, not to check whether it *makes sense during the design process*. Also, I wouldn't say the API is *for* building a consistent, shared mental model. The API is *for* building arrays by copying from another. But a secondary goal is for the two APIs to build a consistent, shared mental model with respect to each other.